tests/tensorflow/requirements.txt

```diff
@@ -1,4 +1,5 @@
 -c ../../constraints.txt
+-r ../../examples/tensorflow/requirements.txt
```

Comment:
You should adjust makefile to avoid the second trying to install the same dependencies. Just to clarify, pre-commit tensorflow tests depend on example requirements?

Reply:
ok removed.
yes